### DISCUSSION TITLE
Tutorial

### DISCUSSION THREAD

**Discussion by HyTheAIGuy** [DataBassGit/BabyBoogaAGI#1] [Q&A]
Could You provide a tutorial on how to setup?

**Comment by DataBassGit** [comment]
Thanks for the feedback. We are in the process of rebuilding the entire app. The current readme is out of date because we have moved away from oobabooga. Their API has been broken for a while now. Currently, we are using the openai API while we finish adding features. Once this is done, we will update the readme with a new installation guide.
